app/src/main/res/values-v21/styles.xml

```diff
@@ -1,15 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
-    <style name="calculatorButton">
+    <style name="CalculatorButton">
         <item name="android:textColor">@color/green</item>
         <item name="android:textSize">@dimen/main_btn_size</item>
         <item name="android:backgroundTint">@color/black</item>
         <item name="android:padding">@dimen/main_btn_padding</item>
+        <item name="android:layout_margin">@dimen/number_btns_margin</item>
+        <item name="android:layout_weight">@integer/layout_weightValue</item>
     </style>
 
-    <style name="calculatorScreen">
+    <style name="CalculatorScreen">
```

Comment:
this name is not representative, include in the name the layout or something more intuitive

Reply:
Done, now its call CalculatorScreenInterface